### DISCUSSION TITLE
tested Arduino MKR Wifi 1010

### DISCUSSION THREAD

**Discussion by RhizomaticNomad** [tttapa/Control-Surface#1041] [General]
As written in the title of this post, I've tested an Arduino MKR Wifi 1010 board. It works well as an MIDI over USB Device, tested with a button as MIDI input. So in my opinion it could be marked as tested in this https://tttapa.github.io/Control-Surface-doc/Doxygen/d8/d4a/md_pages_MIDI-over-USB.html table. Thanks for your work!

**Comment by tttapa** [comment]
Updated. Thanks a lot!
